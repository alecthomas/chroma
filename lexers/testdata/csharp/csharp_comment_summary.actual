/// <summary>
/// Class <c>Point</c> models a point in a two-dimensional plane.
/// </summary>