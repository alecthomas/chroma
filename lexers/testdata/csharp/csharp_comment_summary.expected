[
  {"type":"CommentSpecial","value":"/// \u003csummary\u003e"},
  {"type":"Text","value":"\n"},
  {"type":"CommentSpecial","value":"/// Class \u003cc\u003ePoint\u003c/c\u003e models a point in a two-dimensional plane."},
  {"type":"Text","value":"\n"},
  {"type":"CommentSpecial","value":"/// \u003c/summary\u003e"}
]
